### ISSUE TITLE
Hawaii White Elephant Gift Exchange

### ISSUE THREAD

**Issue by stephaniemcv** [MozillaFoundation/Advocacy#665]
A White Elephant Gift Exchange is a popular holiday game where people vie to walk away with the best present. 

**tldr: I'm asking that each person brings a wrapped gift to Hawaii. Your maximum budget for a gift is $20.**

Quick rundown of the rules: 
1. Each player brings one wrapped gift to contribute to a common pool
2. The first player selects a gift from the pool and opens it
3. The following players can choose to either pick an unwrapped gift from the pool or steal a previous player’s gift. Anyone who gets their gift stolen in this way can do the same – choose a new gift or steal from someone else.

Full rundown of rules, variations and ideas for what makes a great White Elephant Gift here: https://www.whiteelephantrules.com/

To avoid having the TSA make you unwrap your gift and go through their sniffer machine (I learned this the hard way), don't wrap your gift if you are not checking bags! We'll make sure there's some wrapping paper at the hotel.

*Start date: November 28, 2016* 
*Due date: December 8, 2016* 
Note: The dates need to be exactly in the format indicated above for proper integration with the Gantt chart.

CC: @beccaklam @brettgaylor @cadecairos @HPaulJohnson @KevZawacki @lovegushwa @Melechuga @natalieworth @shaghdoosti @ScottDowne @TheoChevalier @vojtechmzl @vanessarhinesmith @WillatMozFdn

**Comment by Melechuga** [comment]
+ @caltrider

**Comment by HPaulJohnson** [comment]
Yes! I'm in. This is awesome.

**Comment by anmechung** [comment]
Great idea!

**Comment by WillatMozFdn** [comment]
Is it allowable under the rules to NOT unwrap your gift once you've chosen it?  :-)

**Comment by stephaniemcv** [comment]
Haha! That might be a fun rule to add!

**Comment by Matt-Holland** [comment]
Everyone's in Hawaii now, so I'm closing this ticket because a) I want to clean up the tickets and b) I'm a party pooper.
